src/member.rs

```diff
@@ -190,6 +191,9 @@ where
     alerts_for_alerter: Option<Sender<Alert<H, D, MK::Signature>>>,
     starting_round_sender: Option<oneshot::Sender<u16>>,
     spawn_handle: SH,
+    newest_counter: NodeCount,
+    starting_round_value: u16,
+    salts: Vec<u64>,
```

Comment:
We don't need multiple hashes, we can reuse one hash which can be stored in a field of Member and initialized in the constructor